### ISSUE TITLE
Rule proposal: no global classes/ids

### ISSUE THREAD

**Issue by marekdedic** [ota-meshi/eslint-plugin-svelte#401]
### Motivation

Hi,
as I am converting an existing codebase to svelte, I encountered some element classes/IDs that no longer do anything have been left over. They were previously used either for stuff like `document.getElementById()` or just global CSS that I over time replaced by component styles.

I thought this could be useful to automate by an eslint rule:

### Description

The rule would disallow any element class or id attributes unless they are used in the component CSS.

### Examples

```svelte
<!-- ✓ GOOD -->
<div />

<!-- ✓ GOOD -->
<div class="my class" />

<style>
.myClass {}
</style>

<!-- ✗ BAD -->
<div class="myClass" />
```

### Additional comments

Does this make sense to you as a rule? I
Also, I'm not sure about the name. Also also, there should probably be a whitelist :).

**Comment by marekdedic** [comment]
Ok, let's start with classes first - I still think IDs make sense as well, but you're right, there are many elements where IDs are use for different things than just styles, so it'd be much more complicated
